### ISSUE TITLE
Standardize release zip files

### ISSUE THREAD

**Issue by jcs090218** [lem-project/setup-lem#7]
In order to make this GHA work, we need to standardize the zip files from the release page:

There are two issues.

#### Linux:

<img src="https://github.com/lem-project/setup-lem/assets/8685505/4121f96d-e455-4eee-9d58-4cc7949984c0" width="70%" />

#### Windows:

<img src="https://github.com/lem-project/setup-lem/assets/8685505/6c639d67-4e81-4d35-a0d7-8edd1a4f8537" width="70%" />

I preferred to organize like Windows, so all files are in the root level. 😉 

### ❓ Issue 2: Extensions are different

The zipped file names are:

- [lem-darwin-v2.0.0.zip](https://github.com/lem-project/lem/releases/download/v2.0.0/lem-darwin-v2.0.0.zip)
- [lem-linux-v2.0.0.tar.gz](https://github.com/lem-project/lem/releases/download/v2.0.0/lem-linux-v2.0.0.tar.gz)
- [lem-windows-v2.0.0.zip](https://github.com/lem-project/lem/releases/download/v2.0.0/lem-windows-v2.0.0.zip)

It's easier to have the same extensions (maybe all `.zip`?). Of course, this is easy to fix in the code, but I would like to acknowledge the Lem's author about this.

cc @cxxxr

**Comment by cxxxr** [comment]
Yes indeed, it may be difficult to handle because it is inconsistent.

About Issue 1, I think either is fine as long as they are unified.

About issue 2, the reason why only linux is tar.gz is because you can't extract the zip unless you put unzip in apt or something.
It might be a good idea to prepare both tar.gz and zip files.

**Comment by jcs090218** [comment]
> About Issue 1, I think either is fine as long as they are unified.

👍 

> About issue 2, the reason why only linux is tar.gz is because you can't extract the zip unless you put unzip in apt or something.
It might be a good idea to prepare both tar.gz and zip files.

It's a good reason! Let me make changes in the code instead, so we don't have to release two versions! 😉
